doc/CLI-usage-guide.md

```diff
@@ -6,11 +8,15 @@ As it is a Java JAR file you need to call it using java:
 ```bash
 java -jar /path/to/kitodo-mediaserver/kitodo-mediaserver-cli.jar
```

Comment:
Shouldn't it say somewhere here, that you may put yaml files such as secrets.yml next to the cli jar? I don't find that documented anywhere (prehaps I haven't searched properly)
If you add it, please also make sure, that the tomcat setup with -Dspring.config.additional-location is documented (I don't find that either)